### PR TITLE
Clean up code in dockertools

### PR DESCRIPTION
**Pull request by yujuhong**
Move functions and sub packages to dockershim.
Part of #43234.

### CODE REVIEW

File: pkg/kubelet/dockertools/docker_manager.go [hunk 1]
```diff
@@ -16,38 +16,8 @@ limitations under the License.
 
 package dockertools
```
Comment (feiskyer) [inline pkg/kubelet/dockertools/docker_manager.go]: Also remove other files inside dockertools?

Comment (yujuhong) [inline pkg/kubelet/dockertools/docker_manager.go]: There will be more PRs to clean up the package. Can't really pack them all in one PR.

**Comment by yujuhong** [comment]
@feiskyer @Random-Liu these are mostly mechanical changes. More PRs will be uploaded to clean up the rest of the files.

**Comment by feiskyer** [comment]
LGTM.

**Pull request merged**